### PR TITLE
Added document for labeling AWS resources on existing clusters

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continues the work in https://github.com/openshift/openshift-docs/pull/5732

Preview Build: http://file.rdu.redhat.com/~ahardin/11082017/rrati-fork/admin_guide/aws_cluster_labeling.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@kwoodson @sdodson @rrati PTAL

I have this appearing in Origin in OCP docs. Please let me know if this should apply to Dedicated docs as well. Thanks!

**Comment by ahardin-rh** [comment]
@kwoodson @sdodson @rrati PTAL and let me know if this should also be applied to the Dedicated doc set. Thanks!

File: admin_guide/aws_cluster_labeling.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+[source,bash]
+----
+$ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
+----
+
+. Restart `atomic-openshift-master` and `atomic-openshift-node` on all nodes:
++
+[source,bash]
+----
+# ssystemctl restart atomic-openshift-master-api atomic-openshift-master-controller atomic-openshift-node
```
Comment (sdodson) [inline admin_guide/aws_cluster_labeling.adoc]: typo here, `systemctl`

**Comment by sdodson** [comment]
> @kwoodson @sdodson @rrati PTAL and let me know if this should also be applied to the Dedicated doc set. Thanks!

It shouldn't be necessary for dedicated.

**Comment by kwoodson** [comment]
@sdodson, @ahardin-rh, I'd imagine we'd want the labeling documentation.  I think we should alert operators of this requirement for dedicated.  Just my .02.

**Comment by sdodson** [comment]
> @sdodson, @ahardin-rh, I'd imagine we'd want the labeling documentation. I think we should alert operators of this requirement for dedicated. Just my .02.

I'd assumed we (Red Hat) were the operators of those environments and dedicated admins needed to take no action of their own?

**Comment by kwoodson** [comment]
@sdodson, Maybe I am misunderstanding the dedicated docs.  Is the operations team the only consumers of those?  Do we have external consumers running dedicated?  If not, then that's fine.

I know that we've been deploying dedicated clusters since before these tags were mandated.  I'm fine if we do not want to doc it for dedicated.  I know there is work to do and we need to go and apply these tags to clusters.

**Review by adellape — APPROVED** [review]
LGTM

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/aws_cluster_labeling.adoc#admin-guide-aws-cluster-labeling[Labeling Clusters for Amazon Web Services (AWS)]
|New topic describing how to label an existing {product-title} cluster running on Amazon Web Services (AWS).
%

**Pull request merged**